libbeat/dashboards/kibana_loader.go

```diff
@@ -48,12 +48,20 @@ func (loader KibanaLoader) ImportIndex(file string) error {
 	params.Set("force", "true") //overwrite the existing dashboards
 
 	// read json file
-	content, err := ioutil.ReadFile(file)
+	reader, err := ioutil.ReadFile(file)
 	if err != nil {
 		return fmt.Errorf("fail to read index-pattern: %v", err)
 	}
 
-	return loader.client.ImportJSON(importAPI, params, bytes.NewBuffer(content))
+	var indexContent common.MapStr
+	err = json.Unmarshal(reader, &indexContent)
+	if err != nil {
+		return fmt.Errorf("fail to unmarshal the index content: %v", err)
```

Comment:
s/fail/failed/ -> not sure which one we shoud use, but we should be consistent :-)

Reply:
I always using `fail` instead of `failed`. Not sure what is more widely used.